### PR TITLE
Fix database/impl androidTest. We need the 22.json schema file, since…

### PR DESCRIPTION
**Pull request by buessow**
… the test tests the migration. The junit-jupiter-api dependency causes errors during build (when the package is merged) and it's not needed for androidTest.

### CODE REVIEW

File: database/impl/schemas/app.aaps.database.impl.AppDatabase/22.json [hunk 1]
```diff
@@ -0,0 +1,3605 @@
+{
```
Comment (MilosKozak) [inline database/impl/schemas/app.aaps.database.impl.AppDatabase/22.json:1]: This file is not necessary

Comment (buessow) [reply database/impl/schemas/app.aaps.database.impl.AppDatabase/22.json:1]: HeartRateDaoTest tests the creation of the heartrate table, i.e. the DB migration from v 22. For that the file is needed.

**Comment by MilosKozak** [comment]
Is it a one-way or two-way data exchange?

**Comment by MilosKozak** [comment]
What is the watchace on the watch?

**Comment by MilosKozak** [comment]
There is already some app for Garmin utilizing dataBroadcaster plugin. Do we need both?

**Comment by MilosKozak** [comment]
As is exchanging data I'd treat it SYNC plugin
Btw superior code quality !!!!

**Comment by buessow** [comment]
> How is it possible that unit tests are not run on this PR?
You mean not run automatically? I don't think androidTests are run automatically.

> Is it a one-way or two-way data exchange?
AAPS sends glucose readings and loop statuses and receive heart rate. So 2 way.

> There is already some app for Garmin utilizing dataBroadcaster plugin. Do we need both?
I think it's tedious for users and development to require an extra app for Garmin integration and it's not too much functionality.

> As is exchanging data I'd make it SYNC plugin Btw superior code quality !!!!
Sync plugin makes sense, I move it. (and thanks :-))

**Comment by buessow** [comment]
I will send another PR to enable androidTest in CircliCI.

**Comment by MilosKozak** [comment]
Sorry i'm not garmin user ... how to use this code on watch then?

**Comment by MilosKozak** [comment]
Every PR runs tests on CircleCI. But PRs from you not. Do you know the reason? I'm a bit afraid to merge to not break something too much

**Comment by buessow** [comment]
I add the watchface to the Garmin app store, once AAPS supports it.
Otherwise, the watch face won't work.

AndroidTests (not only mine) are currently not run by circleci, I'm about
to send a PR to enable that.


Robert Buessow . Mobile: +41 78 899 8296

**Comment by MilosKozak** [comment]
nevermind. i created new PR wih the same commits and unit tests were run
image
I don't know what is different on your PR

**Comment by MilosKozak** [comment]
@buessow what is the name of watchface?

**Pull request merged**